README.md

```diff
@@ -563,6 +563,61 @@ try {
 }
 ```
 
+#### Python: errors in HTTP handlers and `500` errors
```

Comment:
I would also like to add a note about service & data layers not raising HTTP errors (or ApiErrors).